kasa/tapo/tapobulb.py

```diff
@@ -0,0 +1,106 @@
+from typing import Dict, Optional, List
+
+from .tapodevice import TapoDevice
+from ..smartbulb import SmartBulb, HSV, ColorTempRange, SmartBulbPreset
+
+
+class TapoBulb(SmartBulb, TapoDevice):
+
```

Comment:
I wanted to inherit from it to have a common base class to be able to generate/implement the methods from it. The proper solution to avoid mixing implementations is to create a new interface class that is implemented by both `TapoBulb` and `SmartBulb`.

Maybe this is a good time to introduce `Plug(Interface)` and `Bulb(Interface)`.